openshift-knative-operator/cmd/operator/kodata/ingress/1.13/istio/1-net-istio-core.yaml

```diff
@@ -174,6 +174,8 @@ spec:
           capabilities:
             drop:
               - ALL
+          seccompProfile:
```

Comment:
@skonto do we want this? Seems to be in 1.13.

Reply:
One patch is missing from https://github.com/openshift-knative/net-istio/tree/release-v1.13/openshift/patches.
See https://github.com/openshift-knative/net-istio/tree/release-v1.12/openshift/patches. This should not be added.

Reply:
I think this comes from the [same PR](https://github.com/openshift-knative/net-kourier/pull/122), should be fine.

Reply:
Let's see probably its fine.